### ISSUE TITLE
Oozes in The Weeping Cave, Western Plaguelands are way overtuned.

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#3598]
> Oozes in The Weeping Cave, Western Plaguelands are way overtuned.

.go xyz 2260.790000 -2357.920000 60.265700 0 1.148460

**Comment by zana244** [comment]
Checked on internal. Damage per hit seems ok 120-150, however sometimes they cast a Crimson Fury ability which makes them thrash for 3 hits instantly and add fire damage to each hit. Could possibly use a bit of tuning (and checking whether the thrashing is intended)

**Comment by Shadownflame** [comment]
Crimson Fury spell ID=16843 is functionally performing exactly as it does on retail. It gives 3 extra attacks but there is no fire damage despite the visual. Blizzard had likely nerfed that portion of the spell (as in completely removed any fire damage), but forgot to change the tooltip - either way it's a simple Thrash ability on both retail and here. You can see on [this ](https://youtu.be/q36-GgiBd4c?t=1087)classic video that the proc frequency and behavior is matching what we have on Epoch, and the damage is blizzlike as well but Epoch has a small buff in place for every enemy so slightly higher by design.
TL;DR working as intended

PS: this spot is mainly a lvl 60 farm hub for Oozeling and GNPP, so lowbies getting obliterated is a non issue, this place isn't  for them
